psydac/api/fem.py

```diff
@@ -230,61 +246,70 @@ def test_basis(self):
     def trial_basis(self):
         return self._trial_basis
 
+    @property
+    def global_matrices(self):
+        return self._global_matrices
+
     @property
     def args(self):
         return self._args
 
-    def assemble(self, **kwargs):
+    def assemble(self, reset=True, **kwargs):
```

Comment:
For the sake of clarity, I think that we should force the user to pass `reset=True` as keyword-only argument:
```python
def assemble(self, *, reset=True, **kwargs):
```

Reply:
done

Reply:
Thanks